File: lib/firebase/firebase_collections.dart
class FirebaseCollections {
  String get grandPrixesBasicInfo => 'GrandPrixesBasicInfo';
  String get driversPersonalData => 'DriversPersonalData';
  String get teamsBasicInfo => 'TeamsBasicInfo';
  FirebaseSeasonCollections get season => const FirebaseSeasonCollections();
  FirebaseUsersCollections get users => const FirebaseUsersCollections();
}

class FirebaseSeasonCollections {
  const FirebaseSeasonCollections();

  String get main => 'Season';
  String get drivers => 'Drivers';
  String get grandPrixes => 'GrandPrixes';
  String get grandPrixesResults => 'GrandPrixesResults';
  String get teams => 'Teams';
}

class FirebaseUsersCollections {
  const FirebaseUsersCollections();

  String get main => 'Users';
  FirebaseUsersSeasonCollections get season =>
      const FirebaseUsersSeasonCollections();
}

class FirebaseUsersSeasonCollections {
  const FirebaseUsersSeasonCollections();

  String get main => 'Season';
  String get grandPrixesBets => 'GrandPrixBets';
  String get grandPrixesBetPoints => 'GrandPrixBetPoints';
}

File: firebase_functions/firebase_collections.py
class FirebaseSeasonCollections:
    MAIN = "Season"
    DRIVERS = "Drivers"
    GRAND_PRIXES = "GrandPrixes"
    GRAND_PRIXES_RESULTS = "GrandPrixesResults"
    TEAMS = "Teams"


class FirebaseUserSeasonCollections:
    MAIN = "Season"
    GRAND_PRIX_BETS = "GrandPrixBets"
    GRAND_PRIX_BETS_POINTS = "GrandPrixBetPoints"


class FirebaseUsersCollections:
    MAIN = "Users"
    SEASON = FirebaseUserSeasonCollections()


class FirebaseCollections:
    GRAND_PRIXES_BASIC_INFO = "GrandPrixesBasicInfo"
    DRIVERS_PERSONAL_DATA = "DriversPersonalData"
    TEAMS_BASIC_INFO = "TeamsBasicInfo"
    SEASON = FirebaseSeasonCollections()
    USERS = FirebaseUsersCollections()
